ansible/roles/nim-waku/defaults/main.yml

```diff
@@ -74,3 +74,6 @@ nim_waku_websocket_cont_port: 8000
 #nim_waku_websocket_ssl_dir: '/etc/letsencrypt'
 #nim_waku_websocket_ssl_cert: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain }}/fullchain.pem'
 #nim_waku_websocket_ssl_key: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain }}/privkey.pem'
+
+# maximum libp2p connections
+nim_waku_max_connections: 150
```

Comment:
I'd move this close to what it affects, so after port settings.

Also, is this affecting only libp2p or all connections? If only libp2p then I'd rename the variable `nim_waku_p2p_max_conns` so it's clear what it affects.

Reply:
Right. `nim_waku_p2p...` is also closer to the existing naming pattern. Changed in d0cbf21.